packages/common/mui/src/lib/trade-action-header/trade-action-header.tsx

```diff
@@ -42,14 +41,14 @@ export function TradeActionHeader({
       ) : (
         <TokenIcon symbol={token} size="medium" />
       )}
-      <H3
+      <Box
         sx={{
           fontWeight: theme.typography.fontWeightRegular,
           marginLeft: theme.spacing(2),
         }}
       >
         {actionText}
-      </H3>
+      </Box>
```

Comment:
Should this switch back to H3? Does the font size look right here?

Reply:
This was an intentional change. H3 does not look right.